financial-connections/src/main/java/com/stripe/android/financialconnections/appinitializer/FinancialConnectionsInitializer.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.financialconnections.appinitializer
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+import com.airbnb.mvrx.Mavericks
+
+internal class FinancialConnectionsInitializer : ContentProvider() {
+    override fun onCreate(): Boolean {
+        Mavericks.initialize(context = requireNotNull(context))
```

Comment:
👍  I wonder since this is already here if we could start move daggers here too(not needed in this PR)

Reply:
Yeah we actually can!